aeron-system-tests/src/test/java/io/aeron/SpySimulatedConnectionTest.java

```diff
@@ -276,6 +278,35 @@ else if (!isSubscriptionClosed)
         assertEquals(messagesToSend, fragmentCountSpy.value);
     }
 
+    @Test
+    @InterruptAfter(10)
+    void shouldNotHaveShortSendsWithMDCPublication() throws Exception
+    {
+        launch();
+
+        final String channel = "aeron:udp?control=localhost:40456|control-mode=dynamic";
+
+        spy = client.addSubscription(spyForChannel(channel), STREAM_ID);
+        publication = client.addExclusivePublication(channel + "|ssc=true", STREAM_ID);
+
+        Tests.awaitConnected(spy);
+        Tests.awaitConnected(publication);
+
+        Thread.sleep(500);
+
+        AtomicInteger shortSendsCounterId = new AtomicInteger(0);
+        driver.counters().forEach((counterId, typeId, keyBuffer, label) -> {
+            if (AeronCounters.DRIVER_SYSTEM_COUNTER_TYPE_ID == typeId &&
+                SystemCounterDescriptor.SHORT_SENDS.id() == keyBuffer.getInt(0))
+            {
+                shortSendsCounterId.set(counterId);
+            }
+        });
```

Comment:
`Short send` is a system counter and thus it has a fixed id, i.e. just use `SystemCounterDescriptor.SHORT_SENDS.id()` directly.